README.md

```diff
@@ -0,0 +1,9 @@
+AclBundle
+=========
+
+Resources
+---------
+
+  * [Contributing](https://symfony.com/doc/current/contributing/index.html)
```

Comment:
this is wrong, as this bundle is not part of the main repo, so contributing is a bit different